DigitalLearningSolutions.Data.Tests/Helpers/UserTestHelper.cs

```diff
@@ -79,5 +79,10 @@ public static AdminUser GetDefaultAdminUser
                 IsFrameworkDeveloper = isFrameworkDeveloper
             };
         }
+
+        public static string GetDefaultCentreName(string centreName = "North West Boroughs Healthcare NHS Foundation Trust")
```

Comment:
Could this be a variable?

Reply:
Ah whoops I intended to delete that, it's not used any more